general-concepts/dependencies/text.xml

```diff
@@ -421,6 +421,35 @@ RDEPEND="media-libs/cogl:1.0=
 	gnutls? ( &gt;=net-libs/gnutls-2.8:= )"
 </codesample>
 
+<p>
+means that only the '1.0' slot is acceptable for <c>media-libs/cogl</c> and
+that sub-slot changes of <c>media-libs/cogl</c> will cause a rebuild of the
+dependant package. And that every slot of <c>net-libs/gnutls</c> is
+acceptable but any slot change is causing a rebuild.
+</p>
+
+<p>
+The <c>:slot</c> dependency syntax continues to behave like in EAPI=4 or
+earlier, i.e. it indicates that only the specific slot value is acceptable,
+but the package will not break when the version matching the runtime
```

Comment:
If changed the wording to "…and that the package will not break when the currently installed version of the dependency is replaced by a version with a different sub-slot." Since I am not sure what you did not like about the previous version, I can not tell if this addresses your concern. But I hope it does :)